tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -1524,6 +1524,12 @@ public function testBug13310(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug13392(): void
+	{
+		$errors = $this->runAnalyse(__DIR__ . '/nsrt/bug-13392.php');
+		$this->assertNoErrors($errors);
```

Comment:
No need to have this here. The file will be picked up automatically by NodeScopeResolverTest.